ExchangeRateUpdater/ExchangeRateProvider.cs

```diff
@@ -1,10 +1,67 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
+using System.Net;
+using System.IO;
+using System;
 
 namespace ExchangeRateUpdater
 {
     public class ExchangeRateProvider
     {
+        const string URLAddress = "http://www.cnb.cz/cs/financni_trhy/devizovy_trh/kurzy_devizoveho_trhu/denni_kurz.txt";
+
+        //path to file containing raw data
+        string tempFileName;
```

Comment:
Why name it poorly and then attach comment? Would not be something like `string rawDataFilePath` better?